### PR TITLE
Update Rule12-4.md

### PR DESCRIPTION
**Pull request by xingcx**

### CODE REVIEW

**Comment by JasonGlazer** [comment]
Original discussion at the link below. Please give this link time to load. This page might take a few seconds to load and another 4-8 seconds to land at the correct comment:

https://github.com/pnnl/ruleset-checking-tool/pull/19#discussion_r613525487

File: docs/section12/Rule12-4.md [hunk 1]
```diff
@@ -18,19 +18,19 @@
...
+- **Applicability Check 1:** ```sum(length(zone.spaces in U_RMR)) > 0:```
+- Get the name of each space in the building segment in the User model: ```for building_segment_user in U_RMR.building.building_segments:```  
+  - For each thermal_block in building segment: ```thermal_block_user in building_segment_user.thermal_blocks:```
+  - For each zone in thermal block: ```zone_user in thermal_block_user.zones:```
+  - For each space in zone: ```space_user in zone_user.spaces:```  
+    - Get the name of the space: ```space_name_user = space_user.name```
+    - Add to the list of space names in the User model: ```space_names_user_list.append(space_name_user)```  
+
+- Get the name of each space in the building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
+  - For each thermal_block in building segment: ```thermal_block_baseline in building_segment_baseline.thermal_blocks:```
```
Comment (supriyagoel) [inline docs/section12/Rule12-4.md:30]: @xingcx should remove thermal block

File: docs/section12/Rule12-4.md [hunk 2]
```diff
@@ -18,19 +18,19 @@
...
+  - For each zone in thermal block: ```zone_baseline in thermal_block_baseline.zones:```
+  - For each space in zone: ```space_baseline in zone_baseline.spaces:```  
+    - Get the name of the space: ```space_name_baseline = space_baseline.name```
```
Comment (supriyagoel) [inline docs/section12/Rule12-4.md:33]: @xingcx should reference using ID instead of name now.

**Review by supriyagoel — CHANGES_REQUESTED** [review]

File: docs/section12/Rule12-4.md (hunk 1 above)
Comment (xingcx) [reply docs/section12/Rule12-4.md:30]: Updated.

**Comment by mcneillj** [comment]
Archived. No longer needed.

**Pull request closed without merging**